src/studio-home/tabs-section/libraries-tab/index.tsx

```diff
@@ -1,18 +1,115 @@
 import { useIntl } from '@edx/frontend-platform/i18n';
-import { Icon, Row } from '@openedx/paragon';
-import { Error } from '@openedx/paragon/icons';
+import {
+  ActionRow, Form, Icon, Menu, MenuItem, Pagination, Row, SearchField,
+} from '@openedx/paragon';
+import { Error, FilterList } from '@openedx/paragon/icons';
 
 import { LoadingSpinner } from '@src/generic/Loading';
 import AlertMessage from '@src/generic/alert-message';
 import { useLibrariesV1Data } from '@src/studio-home/data/apiHooks';
 import CardItem from '@src/studio-home/card-item';
+import { useCallback, useState } from 'react';
+import SearchFilterWidget from '@src/search-manager/SearchFilterWidget';
 import messages from '../messages';
-import { sortAlphabeticallyArray } from '../utils';
 import { MigrateLegacyLibrariesAlert } from './MigrateLegacyLibrariesAlert';
 
+function findInValues<T extends {}>(arr: T[] | undefined, value: string) {
+  return arr?.filter(o => Object.entries(o).some(entry => String(entry[1]).toLowerCase().includes(
+    String(value).toLowerCase().trim(),
+  )));
+}
```

Comment:
Nit, since we don't use the key here:
```suggestion
function findInValues<T extends {}>(arr: T[] | undefined, searchValue: string) {
  return arr?.filter(o => Object.values(o).some(value => String(value).toLowerCase().includes(
    String(searchValue).toLowerCase().trim(),
  )));
}
```